Tests/UI/DetailsInputViewTests.swift

```diff
@@ -25,7 +25,7 @@ class DetailsInputViewTests: XCTestCase {
     func testSetLabelAndBackgroundColor() {
         let detailsInputView = DetailsInputView()
         detailsInputView.set(label: "addressLine1", backgroundColor: .white)
-        XCTAssertEqual(detailsInputView.label.text, "Address line 1*")
+        XCTAssertEqual(detailsInputView.label.text, "Address line 1")
```

Comment:
Question : there is a very less UI / Unit test for this PR, Are you using old tests to validate new UI ?. Can we add more UI tests

Reply:
Good point, i will add more UI test after new changes.

Reply:
Next iOS meeting, We will discuss UI test code coverage